.github/workflows/main.yml

```diff
@@ -41,8 +41,8 @@ jobs:
           echo Default branch is ${DEFAULT_BRANCH}
           echo Workflow will try to get cache from commit: ${SHA}
 
-          echo ::set-output name=default_branch::${DEFAULT_BRANCH}
-          echo ::set-output name=sha::${SHA}
+          echo "default_branch=${DEFAULT_BRANCH}" >> $GITHUB_OUTPUT
+          echo "sha=${SHA}" >> $GITHUB_OUTPUT
```

Comment:
$GITHUB_OUTPUT echoed somewhere separately?

Reply:
This is new requirement of Github: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/